locale/en_US/submission.xml

```diff
@@ -157,6 +157,93 @@
 	<message key="submission.changeComments">Change to</message>
 	<message key="submission.changeFile">Change File</message>
 	<message key="submission.citations">References</message>
+	<message key="submission.citations.editor.citationlist.approved">Approved</message>
```

Comment:
Are these intentional additions? I don't think they're relevant to this change; perhaps an accidental revert got added to the PR.